formatTest/unit_tests/expected_output/syntax.re

```diff
@@ -1072,3 +1072,35 @@ Some(x: int);
 Some(x, y: int, b);
 
 Some(x, y: int, b);
+
+Some(-1, -1, -1);
+
+Some(-1, -1, -1);
+
+Some(-1g, -1G, -1z);
+
+Some(-1g, -1G, -1z);
+
+Some(-0.1, -0.1, -0.1);
+
+Some(-0.1, -0.1, -0.1);
+
+Some(-0.1G, -0.1x, -0.1H);
+
+Some(-0.1G, -0.1x, -0.1H);
+
+Some([@foo] (-1), [@foo] (-1), [@foo] (-1));
```

Comment:
It's possible to not print the parens in this case, should we remove them?

Reply:
I think so? The fewer the better =D